iconpacks/list.json

```
@@ -33,6 +33,17 @@
       "suffix": "",
       "config": "https://raw.githubusercontent.com/Panniku/vd-iconpacks/master/Packs/MaterialDesign3/config.json",
       "load": "https://raw.githubusercontent.com/Panniku/vd-iconpacks/master/Packs/MaterialDesign3/"
+    },
+    {
+      "id": "Deco",
+      "description": "Deco icons pack for Discord by Icons8",
+      "credits": {
+        "authors": ["icons8", "doraa <455513497288310785>"],
```

```suggestion
        "authors": ["doraa <455513497288310785>"],
```
website doesn't need to be an author as it's already a source